src/lib.jl

```diff
@@ -244,6 +244,38 @@ function sinkhorn2(μ, ν, C, ε; kwargs...)
     return pot.sinkhorn2(μ, ν, PyCall.PyReverseDims(permutedims(C)), ε; kwargs...)
 end
 
+"""
+    empirical_sinkhorn_divergence(X_s, X_t, reg; kwargs...)
```

Comment:
We use other argument names that are more consistent with Julia notation (e.g. lowercase and no underscores if possible):
```suggestion
    empirical_sinkhorn_divergence(xsource, xtarget, ε; kwargs...)
```